src/Stratis.Bitcoin/Base/Deployments/Models/ThresholdStateModel.cs

```diff
@@ -50,6 +51,18 @@ public class ThresholdStateModel
         [JsonProperty(PropertyName = "votes")]
         public int Votes { get; set; }
 
+        /// <summary>
+        /// The total number of blocks in the last confirmation window.
+        /// </summary>
+        [JsonProperty(PropertyName = "blocks")]
+        public int Blocks { get; set; }
+
+        /// <summary>
+        /// A summary of block version counts in the last confirmation window.
+        /// </summary>
+        [JsonProperty(PropertyName = "versions")]
+        public Dictionary<string, int> HexVersions { get; set; }
+
```

Comment:
While you are here can you also add the Period (or MinerConfirmationWindow maybe call it ConfirmationPeroid) it will default to 2016 right now but its a nice to have on the result

Reply:
Good idea.